plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -729,8 +729,12 @@ def update_needed(self, other):
         err_msg = _('Fail to update the %(which)s erratum %(id)s.')
         existing_err_msg = err_msg % {'which': 'existing', 'id': self.errata_id}
         other_err_msg = err_msg % {'which': 'uploaded', 'id': self.errata_id}
-        existing_updated_dt = util.errata_format_to_datetime(self.updated, msg=existing_err_msg)
-        new_updated_dt = util.errata_format_to_datetime(other.updated, msg=other_err_msg)
+        try:
+            existing_updated_dt = util.errata_format_to_datetime(self.updated, msg=existing_err_msg)
+            new_updated_dt = util.errata_format_to_datetime(other.updated, msg=other_err_msg)
+        except ValueError as e:
+            _LOGGER.warn(str(e))
```

Comment:
@mhrivnak that is probably true, although I would have expected the field to be called `created` there. In practice this seems to be an issue for Fedora and EPEL so in my [recap comment below](https://github.com/pulp/pulp_rpm/pull/925#issuecomment-230866214), I am recommending that we do [handle the blank field case more gracefully](https://github.com/pulp/pulp_rpm/pull/925#discussion_r69749973).